UnityProject/Assets/Scripts/Systems/Fluids/PipeFunctions.cs

```diff
@@ -313,8 +323,9 @@ public enum PipeType
 	{
 		None = 0,
 		PipeRun = 1 << 0,
-		CoolingPipe = 1 << 1,
 		//Used for stopping cooling pipes to connect to usual pipes
+		CoolingPipe = 1 << 1,
```

Comment:
I think this is what cooling pipe was meant to be

Reply:
What do you mean by this?

Reply:
Fixed